src/main/java/uk/co/evoco/webdriver/configuration/WebDriverConfig.java

```diff
@@ -91,7 +89,6 @@ public void setWebDriverWaitTimeout(String webDriverWaitTimeout) {
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes